### PR TITLE
Merge in changes from es-interface branch

### PR DESCRIPTION
**Pull request by matthew-andrews**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,9 +6,8 @@ An application that provides an interface for elasticSearch and that polls both
 ## You need the following env vars to be set to run this app
 
 	apikey={FT CAPI key}
```
Comment (i-like-robots) [inline README.md]: Oops, I've been using `API_KEY`

Comment (matthew-andrews) [inline README.md]: nah that's fine.  wanna migrate to `API_KEY` anyway

File: server/lib/get-oldest.js [hunk 2]
```diff
@@ -1,33 +1,42 @@
...
-				}
+	return signedFetch(`https://${CLUSTER}/v${version}_api_v2/item/_search`, {
+			method: 'POST',
+			timeout: 3000,
+			headers: {
+				'Content-Type': 'application/json'
 			},
-			"size": 1
+			body: JSON.stringify({
+				filter : {
```
Comment (i-like-robots) [inline server/lib/get-oldest.js]: I wonder why linting isn't complaining

**Comment by i-like-robots** [comment]
Looks OK I think, easier than rebasing?

**Pull request merged**